h5c/vic/src/vic-webapp/src/app/container-view/container-view.template.html

```diff
@@ -6,18 +6,29 @@
           <clr-alert-item *ngIf="errorObj.type === 'ssl_cert'">
             <span class="alert-text">
               <!-- TODO: i18n -->
-              Failed to verify the VIC machine server at endpoint https://{{ errorObj.payload }}:8443. Your browser may be blocking the connection because of the untrusted SSL certificate.
+              Failed to verify the VIC machine server at endpoint https://{{ errorObj.payload }}:8443. Please check the following:<br/>
```

Comment:
"Please check the following" is a bit confusing; the subsequent list is of possible errors, not things to check. Rephrase one or the other.

Reply:
In that case, "Failed to verify the `vic-machine` server at endpoint https://{{ errorObj.payload }}:8443. Possible causes:"